### PR TITLE
Fix checks

### PR DESCRIPTION
**Pull request by cforgaci**
This PR fixes issues that do not pass `R CMD check`

### CODE REVIEW

File: R/retreive-osm.R [hunk 1]
```diff
@@ -46,7 +46,7 @@ get_osmdata <- function(name, key, value) {
 get_osm_city_boundary <- function(city_name) {
   fetch_boundary <- function(key, value) {
     CRiSp::get_osmdata(city_name, key, value)$osm_multipolygons |>
-      dplyr::filter(dplyr::if_any(c(.data$`name:en`, .data$name),
+      dplyr::filter(dplyr::if_any(c(`name:en`, name),
```
Comment (fnattino) [inline R/retreive-osm.R:49]: This still raises a linter warning:
```
file=R/retreive-osm.R,line=49,col=37,[object_usage_linter] no visible binding for global variable 'name:en'
```
Also a problem in rcoins, not sure what's the best approach to solve it is https://github.com/CityRiverSpaces/rcoins/issues/8

**Review by fnattino — COMMENTED** [review]
Hi @cforgaci, looks good! Still the syntax used in `dplyr` for selection/masking seems to raise warnings with the linter. Not sure what the ideal approach is there.

**Comment by cforgaci** [comment]
@fnattino, I had a look at what is changed here and I see that all issues are solved somewhere else, so I would like to close this PR without merging. Do you agree?

**Comment by fnattino** [comment]
> @fnattino, I had a look at what is changed here and I see that all issues are solved somewhere else, so I would like to close this PR without merging. Do you agree?

Sure, please go ahead!

**Pull request closed without merging**